catalog/solr/catalog-solr-core/src/main/java/ddf/catalog/source/solr/SolrMetacardClientImpl.java

```diff
@@ -938,4 +981,18 @@ private org.apache.solr.client.solrj.response.UpdateResponse softCommit(
             /* softCommit */ true)
         .process(client.getClient());
   }
+
+  private ddf.catalog.operation.QueryResponse putMetricsDuration(
+      ddf.catalog.operation.QueryResponse response, String key, long value) {
+    putMetricsDuration(response.getRequest(), key, value);
```

Comment:
❓ Similar question here: is putting the properties on the request better than on the response?  Seems more appropriate information in the response properties.

Reply:
see previous response